src/main/java/com/nerodesk/Launch.java

```diff
@@ -61,9 +78,22 @@
      * @throws IOException If something goes wrong.
      */
     public Launch(final int port) throws IOException {
+        this(port, new MkStorage());
+    }
+
+    /**
+     * Constructor with port and Store.
+     * @param port Port.
+     * @param store Store to start with.
+     * @throws IOException If something goes wrong.
+     */
+    public Launch(final int port, final Storage store) throws IOException {
         Logger.info(Launch.class, "HTTP server starting on port %d", port);
         new FtBasic(
-            new TsRegex().with("/", new TkIndex()),
+            new TsFork(
+                new FkRegex("/", new TkIndex()),
+                new FkRegex(TkGetFile.PATH, new TkGetFile(store))
```

Comment:
We need to use this path in ```TkGetFileTest```, so it can't be inlined (otherwise it would be duplicated)